DATA/FX/effects.ini

```diff
@@ -1520,10 +1520,26 @@ effect_type = EFT_WEAPON_IMPACT
 vis_effect = pi_particle_03_impact
 
 [Effect]
-nickname = pi_particle_03_proj

```

Comment:
pi_particle_03_proj is exchanged with pi_particle_05_flash (pi_particle_03_proj not there anymore)